services/src/main/webapp/WEB-INF/access-fedora-context.xml

```diff
@@ -42,11 +42,17 @@
         <property name="objectAclFactory" ref="objectAclFactory" />
     </bean>
     
+    <bean id="repositoryObjectLoaderNoCache" class="edu.unc.lib.dl.fcrepo4.RepositoryObjectLoader" init-method="init">
```

Comment:
Might be able to refactor this out, and explicitly ask for refreshed copies of objects